Controls/Input/Input.UWP/Themes/Calendar.xaml

```diff
@@ -264,7 +263,41 @@
             </Setter.Value>
         </Setter>
     </Style>
+    
+    <Style TargetType="Button" x:Key="CalendarFooterControlButtonStyle">
+        <Setter Property="Width" Value="47"/>
+        <Setter Property="Content" Value="+"/>
+        <Setter Property="HorizontalAlignment" Value="Right"/>
+        <Setter Property="FontSize" Value="24"/>
+        <Setter Property="Foreground" Value="{ThemeResource TelerikCalendarFooterButtonForegroundBrush}" />
+        <Setter Property="Background" Value="Transparent"/>
+    </Style>
 
+    <Style TargetType="localCalendar:CalendarFooterControl" x:Key="CalendarFooterControlStyle">
+        <Setter Property="FooterControlButtonStyle">
+            <Setter.Value>
+                <Style BasedOn="{StaticResource CalendarFooterControlButtonStyle}" TargetType="Button"/>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Height" Value="48"/>
+        <Setter Property="IsTabStop" Value="False" />
```

Comment:
Why tap navigation is stopped?

Reply:
Same as navigation control - the tab will be redirected to the button within the FooterControl instead.